packages/terra-worklist-data-grid/src/subcomponents/ColumnHeaderCell.jsx

```diff
@@ -183,19 +187,28 @@ const ColumnHeaderCell = (props) => {
   // Retrieve current theme from context
   const theme = useContext(ThemeContext);
 
+  const pinnedColumnsDivider = columnIndex === columnContext.pinnedColumnsLength - 1
+    ? (
+      <div
+        className={cx('pinned-columns-divider')}
+        style={{ height: tableHeight, left: width - 1 }} // eslint-disable-line react/forbid-dom-props
+      />
+    )
+    : null;
```

Comment:
It looks like you should be able to add this directly into the JSX of the return statement using conditional rendering.  You don't need this additional variable

Reply:
Right now using a variable clarifies the purpose of this div (that it is a pinned columns divider). Also, having a ternary operation within the return statement reduces readability and makes it more difficult to understand at a glance.